### PR TITLE
docs(release-notes): 1.3 release notes

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [x] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [x] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

Updated release-notes.md to have 1.3 changes

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -1,45 +1,41 @@
...
 
 ### Additions
-#### [RPS](Glossary.md#r)
-- **Hostname and Device Metadata tags:** We have added the ability to define metadata tags for a each Intel&reg; AMT profile.  These tags will be applied to each device that is configured using the profile.  Additionally, if RPC sends up the Intel&reg; AMT device OS hostname during activation, RPS will save this information as part of the device information.  When RPS configures a device, it will perform a POST call to MPS to register that device in MPS.  This works in conjunction with the improved metadata gathering that RPC added in 1.1.
-- **Node 14 LTS support:** RPS has been updated to support running on Node 14 LTS. 
+#### RPS & MPS
+- **Stateless Authentication:** The RESTful APIs for both RPS and MPS now utilizie stateless authentication tokens to validate user access.  With this change, our reference deployment now utilizes Kong API Proxy that validates the token and then fowards authenticated requests to either MPS or RPS.  This simplifies how API calls are made into the Open AMT Cloud Toolkit services by putting the service name that is being called into the URL.
```
Comment (rsdmike) [inline docs/release-notes.md:10]: utilizie => utilize

Comment (rsdmike) [reply docs/release-notes.md:10]: The first sentence I can't get past. You clarify in the second sentence that KONG is doing the validation rendering the first sentence false. So I'd  say something like , the Restful APIs for both RPS and MPS have had their APIKEYS removed and now leverage KONG for authentication with JWT Tokens.  -- Anywhere you say token i'd say JWT token specifically.

File: docs/release-notes.md [hunk 2]
```diff
@@ -1,45 +1,41 @@
...
 
-#### [RPC](Glossary.md#r)
-- **Docker support added:** Added a dockerfile to RPC that allows customers to build and execute RPC in a docker container on the Intel&reg; AMT device.
+    ```html
+    https://example.url/mps
+    https://example.url/rps
+    ```
+    
+    With the addition of Kong and its rich plug-in support, this opens up options for integrators to utilize a broader set of user authentication services.  For example, the same tokens that are already being used by the rest of a device management tool to authenticate users can also be used to authenticate users with the Open AMT Cloud Toolkit service(s).  This new feature should make integration of Open AMT Cloud Toolkit microservices into existing management consoles much more seamless.
```
Comment (rsdmike) [inline docs/release-notes.md:17]: I'd link to the list of auth plugins for folks to see what can be leveraged. And additionally call out that MPS `/authorization` endpoint now provides a JWT token that can be used.

File: docs/release-notes.md [hunk 3]
```diff
@@ -1,45 +1,41 @@
...
 
+### Modifications and Removals
+#### RPS
+- **RPS Activation Changes:** In 1.2 we added a short delay between the time we activated an AMT device and when we configured the AMT features.  While RPS waiting for AMT to be ready, RPS and RPC would send "heartbeats" back and forth to keep the connection alive.  This "heartbeat" rate was very fast and would cause thousands of progress dots to be output to the client terminal screen while this was happening.  In 1.3 we have fixed this "heartbeat" pace to only 1 "heartbeat" every 5 seconds.  Many dots (not to mention packets) were saved with this change.
```
Comment (rsdmike) [inline docs/release-notes.md:21]: While RPS **is** waiting for AMT to be ready....

Comment (rsdmike) [reply docs/release-notes.md:21]: Mention disk space saved due to log  output spam?

File: docs/release-notes.md [hunk 4]
```diff
@@ -1,45 +1,41 @@
...
-#### [MPS](Glossary.md#m) & [RPS](Glossary.md#r)
-- **Deprecation announcement of Developer Mode for both RPS and MPS** We are making strides in improving production mode deployments through the use of deployment scripts and docker containers.  Due to this work, the need for a "simple" developer deployment mode that doesn't use Vault or PostgreSQL is no longer required.  As such, we'll be removing this mode prior to our 1.4 release.  After the removal of developer mode, Vault and PostgreSQL will be required by default but the docker deployment scripts we have will make getting these services up and running easy.  Customers will still have the option to modify the Vault and PostgreSQL interfaces to point to the database service(s) of their choice.
 
 ## Resolved Issues
+- **KVM freeze intermittently:** We have added a small delay in handling mouse interactions that prevents us from flooding the AMT channel.  There are still a few occasions where KVM could still freeze.  We are still root causing this new issue
```
Comment (rsdmike) [inline docs/release-notes.md:33]: Don't feel like this can go here if we are still root causing

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: docs/release-notes.md (hunk 2 above)
Comment (kimberlyxdavis) [inline docs/release-notes.md:17]: Stateless Authentication with Kong API Proxy: The RESTful APIs for both RPS and MPS now support the use of stateless authentication tokens to validate user access. The reference deployment uses Kong API Proxy to validate the token and forward authenticated requests to either MPS or RPS. This change simplifies calls to the Open AMT Cloud Toolkit services by inserting the service name that is being called into the URL.
Format:
https://example.url/<servicename>
Examples:
https://example.url/mps
https://example.url/rps
Kong, with its rich plug-in support, enables integrators to use a broader set of user authentication services. For example, the same tokens that are already being used by the rest of a device management tool to authenticate users can also be used to authenticate users with the Open AMT Cloud Toolkit service(s). This new feature improves the experience of integration of Open AMT Cloud Toolkit microservices into existing management consoles.

File: docs/release-notes.md (hunk 3 above)
Comment (kimberlyxdavis) [inline docs/release-notes.md:21]: - **RPS Activation Changes:** In 1.2 we added a short delay between the activation of a device and the configuration of AMT features.  While the RPS waited for AMT to be ready, RPS and RPC sent "heartbeats" back and forth to keep the connection alive.  This "heartbeat" rate was very fast and produced thousands of progress dots on the client terminal screen.  In 1.3 we have modified this "heartbeat" pace to send a heartbeat every five seconds.  Many dots (not to mention packets) were saved with this change.

File: docs/release-notes.md [hunk 5]
```diff
@@ -1,45 +1,41 @@
...
+#### UI Toolkit
+- **Deprecation announcement of non-socket UI Toolkit components:** Following the deprecation announcement in 1.2, we have removed the following UI Toolkit components: Profile Editor, CIRA Editor, Domain Editor, Audit Log, and Device List.  KVM and SOL UI Toolkit modules will remain in the UI Toolkit.  Look for broader UI framework support coming in future releases.
+#### MPS & RPS
+- **Removal of Developer Mode:** As was announced in 1.2 release, we removed "Developer Mode".  In "Developer Mode" our services utilized a file-based approach to storing data.  Operating in this mode had several potentially serious issues from security of the data stored in the files, scaling performance, not to mention the additional code required to support reading from a file versus accessing an API.  With this change, the MPS and RPS will exclusively utilize the DB and Vault interfaces for storing and accessing data.  The recommended deployment model going forward is to utilize containers (docker).
```
Comment (kimberlyxdavis) [inline docs/release-notes.md:25]: I'd like to suggest that we ditch utilize. It feels like an obfuscation. I like (and most style manuals support this)  "uses",  "supports", "enables", "provides", "incorporates", "integrates", "offers", etc.  The words that we choose from that list clarify what is happening just a tiny bit more.

File: docs/release-notes.md [hunk 6]
```diff
@@ -1,45 +1,41 @@
...
+- **API refactoring is complete:** In 1.2 we refactored the RPS REST APIs and in 1.3 we performed a similar refactoring the MPS REST APIs.  With the release of 1.3, these API breaking changes have been completed.  Special note: with the addition of Stateless Authenication, the API key in the header has been removed.  Please take a look at our updated Swagger documentation:
```
Comment (kimberlyxdavis) [inline docs/release-notes.md:26]: What is the big take-away with the refactoring? Is it that the code became simpler, more legible?

File: docs/release-notes.md (hunk 1 above)
Comment (kimberlyxdavis) [reply docs/release-notes.md:10]: Yeah, I struggled here as well. I tried to rewrite in the comment, but I fear I may have changed the meaning (or not).

Comment (matt-primrose) [reply docs/release-notes.md:10]: Rephrased this section

File: docs/release-notes.md (hunk 2 above)
Comment (matt-primrose) [reply docs/release-notes.md:17]: Updated with links to Kong's Plug-in Hub and added reference to MPS /authorization endpoint

File: docs/release-notes.md (hunk 3 above)
Comment (matt-primrose) [reply docs/release-notes.md:21]: Done

File: docs/release-notes.md (hunk 4 above)
Comment (matt-primrose) [reply docs/release-notes.md:33]: Back to the known issue list it goes!

File: docs/release-notes.md (hunk 2 above)
Comment (matt-primrose) [reply docs/release-notes.md:17]: I've re-worked this area.  Please review and see if it works better

File: docs/release-notes.md (hunk 3 above)
Comment (matt-primrose) [reply docs/release-notes.md:21]: Updated

File: docs/release-notes.md (hunk 5 above)
Comment (matt-primrose) [reply docs/release-notes.md:25]: Updated

File: docs/release-notes.md (hunk 6 above)
Comment (matt-primrose) [reply docs/release-notes.md:26]: good point, I'll add a take away sentence in here as well.

**Review by kimberlyxdavis — COMMENTED** [review]
While it's not always how I would things, I think it has improved. The release notes seem to be trending toward a more casual construction.

**Review by kimberlyxdavis — APPROVED** [review]

**Pull request merged**